besu-plugins/linea-sequencer/sequencer/src/main/java/net/consensys/linea/sequencer/txpoolvalidation/LineaTransactionPoolValidatorFactory.java

```
@@ -72,6 +77,7 @@ public LineaTransactionPoolValidatorFactory(
   public PluginTransactionPoolValidator createTransactionValidator() {
     final var validators =
         new PluginTransactionPoolValidator[] {
+          new TraceLineLimitValidator(invalidTransactionByLineCountCache),
```

good point making this the first validation in the chain, so it fail fast